code/modules/mob/living/simple_animal/constructs/constructs.dm

```diff
@@ -233,6 +237,7 @@
 	resistance = 10
 	var/energy = 0
 	var/max_energy = 1000
+	can_climb = 1 //It flies, its magic
```

Comment:
Holy shit.

I do not need a Hulk-like Super buff invading engineering through the T-Comms ladders.

100% Agree with LordFowl and Alberyk